vtex/faststore
# feat: Adds `Navbar` component

## What's the purpose of this pull request?

`faststore/components`
- [x] Moves `Navbar` component;
- [x] Moves `NavbarSlider` component;
- [x] Moves `NavbarLinks` component;
- [x] Adds `Navbar` doc;
- [x] Adds `NavbarSlider` doc;
- [x] Adds `NavbarLinks` doc;
- [x] Adds `Navigation` feature doc;

`faststore/ui`
- [x] Adds `Navbar` styles;
- [x] Adds `NavbarSlider` styles;
- [x] Adds `NavbarLinks` styles; 

`faststore/core`
- [x] Review `Navbar`, `NavbarSlider` and `NavbarLinks` structures;
- [x] Delete stories and styles;

## How to test it?

1. Check the [Navigation](https://faststore-site-pxtkhfknl-faststore.vercel.app/components/features/navigation) feature and all the related components to that feature (`Navbar`, `NavbarSlider` and `NavbarLinks` and its compound components.

2. run `yarn dev` on `faststore/core` application and check the `Navbar` on the homepage.

### Starters Deploy Preview
